### PR TITLE
Issue #13213: Remove '//ok' comments from equalshashcode

### PR DESCRIPTION
**Pull request by biplavoraon**
Related to #13213 

Removed all the '//ok' comments from EqualsHashCode.

In InputEqualsHashCodeSemantic.java file, the following 2 comments are left as it is:
1. // wrong arg type, ok
2. // wrong arg type, ok even with generics

### CODE REVIEW

**Review by MANISH-K-07 — COMMENTED** [review]
The failing CI seem unrelated to the changes made in this commit.
Please mark this PR ready for review so maintainers can have a look

**Review by romani — APPROVED** [review]
ok to merge if CI is green

**Pull request merged**
